bears/vcs/git/GitCommitBear.py

```diff
@@ -204,3 +224,81 @@ def check_body(self, body,
             yield Result(self, 'Body of HEAD commit contains too long lines. '
                                'Commit body lines should not exceed {} '
                                'characters.'.format(body_line_length))
+
+    def check_issue(self, body,
+                    detect_host: bool=False,
+                    require_url: str=None,
+                    keyword_regexp: str=None):
+        """
+        Check for matching issue related URLs.
+
+        :param body:                Body of the commit message of HEAD.
+        :param detect_host:         Whether to automatically detect host
+                                    from git config or not.
+        :param require_url:         Whether the commit message requires
+                                    the presence of URL or not. 'full_ref'
+                                    confines commit to have full issue URL.
+                                    'short_ref' confines commit to have
+                                    shorter issue references like #123, etc.
+        :param keyword_regexp:      If provided, matches the keywords
+                                    with the regex. This gains precedence
+                                    over auto_detect_host and require_url.
```

Comment:
Docstring is not correctly aligned, either do
```
:param xyz: Hello
:param x:    Hello2
```
or (what I recommend as it eases maintenance of long docstrings):
```
:param xyz:
    Hello
:param x:
    Hello2
```